src/test/rustdoc/intra-doc/prim-self.rs

```diff
@@ -5,6 +5,7 @@
 #![feature(no_core)]
```

Comment:
For some reason I had to modify this test after applying @camelid's [suggestion](https://github.com/rust-lang/rust/pull/95316#discussion_r847625546)

Reply:
Hmm, what was the test failure?

Reply:
`.value()` uses the const machinery, so I bet now it's looking for lang items that it didn't need before. The test diff looks correct by the way.